logbook-api/src/main/java/org/zalando/logbook/LogbookCreator.java

```diff
@@ -38,6 +39,10 @@ private static Logbook create(
                 .reduce(QueryFilter::merge)
                 .orElse(null);
 
+        @Nullable final PathFilter pathFilter = pathFilters.stream()
```

Comment:
I'm a big fan of the [Open-closed principle](https://en.wikipedia.org/wiki/Open%E2%80%93closed_principle). What is the benefit of introducing a new filter type? This could be implemented as a specialized request and response filter, right?

Reply:
The primary benefit is being more developer-friendly. I am pretty sure it is more intuitive for developers with a path-filter when there also is a header, body and query filter; these four are the essential parts of HTTP after all. Also a working filter implementation will save time. 

So while much can be done with the RequestFilter and ResponseFilter, those are more 'catch all' in my view.

Reply:
Fair enough.